policyengine/countries/uk/default_reform/reform.py

```diff
@@ -619,8 +619,10 @@ def formula(household, period, parameters):
             person = household.members
             tax_band = person("tax_band", period)
             bands = tax_band.possible_values
-            eligible = ~household.any(
-                (tax_band == bands.ADDITIONAL) | (tax_band == bands.HIGHER)
+            eligible = household.any(
+                (tax_band == bands.BASIC)
+                | (tax_band == bands.STARTER)
+                | (tax_band == bands.INTERMEDIATE)
             ) * (household("smf_benefit_cash_payment", period) == 0)
```

Comment:
Actually could we change this logic so it works even with there's no benefit parameter set? That is, use eligibility for it.

Reply:
Yep on it